ci-operator/step-registry/mtr/deploy-windup/README.md

```diff
@@ -0,0 +1,33 @@
+# mtr-deploy-windup-ref<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Purpose](#purpose)
+- [Process](#process)
+- [Requirements](#requirements)
+  - [Infrastructure](#infrastructure)
+  - [Variables](#variables)
+
+## Purpose
+
+To deploy Windup in a specified namespace with a specified volume capacity. This step is necessary to execute the interop tests for the MTR operator.
+
+## Process
+
+This script is very simple. It uses `oc` to deploy Windup, then checks continuously checks if the deployment is complete and ready. If it isn't ready 
+## Requirements
+
+### Infrastructure
```

Comment:
Is there any way to find out the CPU/memory requirements? That will help define the CRC resources when testing
locally

Reply:
We could add that to the README, but the ref's yaml file does have the `resources` section. @ascerra how do you feel about adding a physical resources section to these documents?

Reply:
We can add that, but the step's yaml file does have a stanza for that. @ascerra how do you feel about a "Physical Resources" section of the README documents?

Reply:
@Simranpal -

@ascerra and I discussed this in a 1:1 we had today and both thought we could leave any CPU/Memory requirements out of the README documents as they are already defined in the step's yaml file. 

We think the one exception to that rule is if the CPU and Memory requirements were really high. In that case, we can put it under the `other` header in the `requirements` section of the READMEs to explain why so many resources would be needed.

Reply:
Alright sounds good!